src/webextension/ticker-timer-modes.js

```diff
@@ -89,8 +92,12 @@ async function set_ticker_update_function() {
 // For updating the time shown in the popup ticker. Returns a promise/string.
 async function get_popup_ticker_default() {
     try {
-        let url = await get_current_url(),
-            domain = url.host,
+        let url = await get_current_url();
+        if (!url) {
+            // e.g. incognito mode (should be unreachable, popup is disabled)
+            return '???';
```

Comment:
Maybe using a special url here (like with blue-mode) would be better than using null, since it would differentiate private browsing from other possible cases where the url might be null or undefined.  (To tell the two special urls apart, maybe we could include the word "private" in the url before the series of random characters.)  Then we can return a formatted string like "Private / 0:12" as with the other possible return values from this function.  (There's also the question of how to handle private browsing in blue mode.  I'll think some more about that.)